docs/self-hosting/methods/download-config.md

```
@@ -0,0 +1,197 @@
+const airgappedResponse = await fetch(
+  "https://prime.plane.so/api/v2/setup/?version=v0.23.0&airgapped=true&platform=arm64"
+);
+const airgappedBlob = await airgappedResponse.blob();
+```
+
+</template>
+</CodePanel>
+
+<ResponsePanel status="200">
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)